### ISSUE TITLE
【CSP once is not effective 】util/src/Dom/dynamicCSS injectCSS function with csp bug,Style.once is not effective

### ISSUE THREAD

**Issue by shentao1** [react-component/util#495]
util/src/Dom/dynamicCSS injectCSS function ，Style.once only assigns attributes and is still blocked by CSP policies. Style.setAttribute should be used to set nonce to the dom tree.
![image](https://github.com/react-component/util/assets/20812348/e6a25fd5-ddfa-4a89-a722-065bc22c6944)
The following example also illustrates that using. xx assignment does not reflect on the dom tree, and using setAttribute will reflect on the dom tree
![image](https://github.com/react-component/util/assets/20812348/3b16b0d7-9a41-43e7-991e-a03da03e5a9d)

**Comment by shentao1** [comment]
![image](https://github.com/react-component/util/assets/20812348/148defd6-3010-411e-a487-fcc2e4aaf8e5)
This bug also exists here

**Comment by dVerhees** [comment]
I believe this is a duplicate of https://github.com/react-component/util/issues/392
